main.go

```diff
@@ -49,6 +57,31 @@ func startRun(config *common.CatalogConfig, rh request.Handler) {
 	rh.StartHandlingRequests(config, &towerapiworker.DefaultAPIWorker{})
 }
 
+func getConfigFile() (string, error) {
+	configFile := "./rhc/workers/catalog.toml"
+	if fileExists(configFile) {
+		return configFile, nil
+	}
+	usr, err := user.Current()
```

Comment:
@bzwei we need to handle this err, if we cannot determine the user for any reason.

Reply:
@bzwei would it make sense to create an array of the filenames and then iterate thru the list till we get the first one
```
func getConfigFile() (string, error) {
filename, _ := range configFiles() {
    if fileExists(filename) {
         return filename, nil
    }
}
    return nil, fmt.Errorf("No config file found")
}
```

Reply:
My assumption is that the login does not have a user profile, then we should proceed to detect the cofig file under /etc/. If we handle this error and stop here, we'd have no chance to go on.

We could simply log the error but continue with next try. What do you thinik?

Reply:
My browser did not auto refresh. I missed you above comment. I will work on it.